Dynavity/Dynavity/view/MainView.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct MainView: View {
+    @ObservedObject private var canvasViewModel = CanvasViewModel()
```

Comment:
Should the `CanvasViewModel` be initialized in each `CanvasVIew` instead? Since we would likely have multiple canvases in a `MainView`, so each `CanvasView` would have their own `CanvasViewModel` instead of sharing a common one from `MainView`?

Reply:
hmm by `MainView`, I'm thinking of the view that the user would spend the most time on, which is the view where the user can modify the canvas. So `MainView` contains only 1 canvas, unless the intention is for the user to have multiple canvases open at the same time?

Reply:
I see! Alright if `MainView` contains only 1 canvas, then I think its good!